modules/oe_content_news/config/post_updates/00007_update_view_display/core.entity_view_display.node.oe_news.default.yml

```diff
@@ -0,0 +1,76 @@
+langcode: en
```

Comment:
we don't update default view mode in oe_content. It has to be done in oe_theme.

Reply:
To show the Contacts field on news, without using oe_theme, we need to update the default view for the content type. Should we not show Contacts field when not using oe_theme?

Reply:
If oe_theme isn't used it means that custom default view mode is used on the site. We shouldn't override it.

Reply:
Update removed